### ISSUE TITLE
Droplets on windshield on a solid runway

### ISSUE THREAD

**Issue by gilbertohasnofb** [Juanvvc/c172p-detailed#581]
Try this: load the sim with the amphibious on water (or land it there) so droplets will appear on windshield. Then go to the move to airport dialog and select a solid runway. The windshield will remain with the droplets:

![fgfs-screen-001](https://cloud.githubusercontent.com/assets/5700795/11451908/e3f156a4-95d5-11e5-9eab-49cd1723fda5.png)

@wlbragg can you check this out?

**Comment by wlbragg** [comment]
Will do.

**Comment by wlbragg** [comment]
I just tried this and it didn't leave any drops on my windshield when changing to another airport after landing on water?

**Comment by wlbragg** [comment]
Have you done this multiple times and it is consistent?

**Comment by wlbragg** [comment]
We are using to trigger this, can hydro_active_norm be > 0 on land?
~~~xml
if (getprop("position/altitude-agl-m") < 2 and hydro_active_norm > 0)
~~~

**Comment by gilbertohasnofb** [comment]
> Have you done this multiple times and it is consistent?

Not really, I just spot this while doing some other tests. I will try to see if I can reproduce it and then report back.

**Comment by gilbertohasnofb** [comment]
I am trying to reproduce it, but I can't :cry: The closer to the initial report I got was when I started the sim with the amphibian in a port (SP01), speed up so the droplets start to run fast through the windshield, then went to "Go to Airport" and selected KSFO. For some moments the droplets were there, but then they disappear. 

Anyway, if no one can reproduce this, we can as well close it. If in the future we come across this bug then we reopen and work on it.
